src/server/cluster/cluster_family.cc

```diff
@@ -245,12 +214,56 @@ void ClusterFamily::ClusterSlots(ConnectionContext* cntx) {
   }
 }
 
+namespace {
+void ClusterNodesImpl(const ClusterShards& config, string_view my_id, ConnectionContext* cntx) {
+  // For more details https://redis.io/commands/cluster-slots/
+
+  string result;
+
+  auto WriteNode = [&](const Node& node, string_view role, string_view master_id,
+                       const vector<SlotRange>& ranges) {
+    absl::StrAppend(&result, node.id, " ");
+
+    absl::StrAppend(&result, node.ip, ":", node.port, "@", node.port, " ");
+
+    if (my_id == node.id) {
+      absl::StrAppend(&result, "myself,");
+    }
+    absl::StrAppend(&result, role, " ");
+
+    absl::StrAppend(&result, master_id, " ");
+
+    absl::StrAppend(&result, "0 0 0 connected");
+
+    for (const auto& range : ranges) {
```

Comment:
I think that only master prints the slots range

Reply:
Wow, nice catch. I couldn't even find this in Redis' documentation, but it is aligned with their provided examples. Anyway, fixed.